# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: js/src/wasm/WasmBCClass.h
# Commit: 3ecda60613c5
# Full Hash: 3ecda60613c5cb7a57cc7004919c7d12c3a29268
# Author: Lars T Hansen <lhansen@mozilla.com>
# Date: 2021-10-22 03:08:27
# Description:
#   Bug 1736531 - Fix. r=jseward
#   
#   See bug for analysis.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D129010
# ==============================================================================

diff -r 1044b04cf89b -r 3ecda60613c5 js/src/wasm/WasmBCClass.h
--- a/js/src/wasm/WasmBCClass.h	Thu Oct 21 15:44:59 2021 +0000
+++ b/js/src/wasm/WasmBCClass.h	Thu Oct 21 15:46:28 2021 +0000
@@ -794,8 +794,8 @@
   // hot enough to warrant inlining at the outermost level.
 
   inline void needResultRegisters(ResultType type, ResultRegKind which);
-#ifdef JS_CODEGEN_X64
-  inline void maskResultRegisters(ResultType type);
+#ifdef JS_64BIT
+  inline void widenInt32ResultRegisters(ResultType type);
 #endif
   inline void freeResultRegisters(ResultType type, ResultRegKind which);
   inline void needIntegerResultRegisters(ResultType type);